### PR TITLE
BaSP-MR-86: Inputs error and nomarl input with placeholder

### PR DESCRIPTION
**Pull request by Emanuel-Lamberti**
The global inputs were designed where it shows me an error (with a red border and text), and another where a text can be seen inside the input (placeholder).

### CODE REVIEW

**Review by FrancoLelli — APPROVED** [review]

File: src/Components/Shared/Inputs/InputError/index.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  const handleBlur = () => {
+    setFocus(false);
+  };
+
+  const inputClassName = `input ${error && !focus ? 'error' : ''}`;
+
+  return (
+    <div className="input-container">
+      <label>{placeholder}</label>
```
Comment (FrancoLelli) [inline src/Components/Shared/Inputs/InputError/index.jsx:19]:
```suggestion
```

**Review by FrancoLelli — CHANGES_REQUESTED** [review]
Delete label

File: src/Components/Shared/Inputs/InputError/error-placeholder-input.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.input-container input {
+  width: 260px;
+  height: 20px;
+  border-radius: 4px;
+  padding: 10px 12px 10px 12px;
+  border-color: #373867;
+}
+
+.input-container label {
```
Comment (FrancoLelli) [inline src/Components/Shared/Inputs/InputError/error-placeholder-input.css:9]:
```suggestion

```

File: src/Components/Shared/Inputs/InputPlaceholder/placeholder-input.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
+.inputPlaceholder-container input {
+  width: 260px;
+  height: 20px;
+  border-radius: 4px;
+  padding: 10px 12px 10px 12px;
+  border-color: #373867;
+}
+
+.inputPlaceholder-container label {
```
Comment (FrancoLelli) [inline src/Components/Shared/Inputs/InputPlaceholder/placeholder-input.css:9]:
```suggestion

```

**Review by FrancoLelli — CHANGES_REQUESTED** [review]
Delete styles, because you dont use a label

**Review by CristianLotorto — APPROVED** [review]
Nice job. Fix the comments and it's all set

**Review by CristianLotorto — APPROVED** [review]
Nice job. Fix the comments and it's all set

**Comment by Emanuel-Lamberti** [comment]
Label removed.

**Review by FrancoLelli — APPROVED** [review]

**Pull request merged**